src/app/components/mnoe-product-markups-list/mnoe-product-markups-list.coffee

```diff
@@ -69,6 +71,12 @@
         (response) ->
           vm.markups.totalItems = response.headers('x-total-count')
           vm.markups.list = response.data
+          if vm.customerOrg
+            vm.readOnlyView = true
+            vm.markups.list = _.filter(vm.markups.list, (app) ->
+              app if !app.product ||
+              _.includes(_.map(vm.customerOrg.active_apps, 'nid'), app.product.nid)
```

Comment:
The `_.map(vm.customerOrg.active_apps, 'nid')` could be extracted outside of the `filter` loop to avoid recomputing it at each iteration